### DISCUSSION TITLE
Pictures to currency calculator

### DISCUSSION THREAD

**Discussion by D3nan** [D3nan/Tafe24S1T102_Ext#12] [Q&A] [answered]
In the images folder, there are pictures of the various flags of the countries. Could add this to the drop down box or in the final output

**Comment by SamuelTAFE** [comment, marked as answer]
Hi Denan,
That's a great idea for a future feature.
We could even try and get the image to display next to the appropriate currency, at the same time that the currency conversion calculation is done.
